content/docs/0.16.x/manage/shipyard/index.md

```diff
@@ -1,290 +1,103 @@
 ---
-title: Shipyard
+title: Working with shipyard files
 description: Information about shipyard, sequences and tasks to define processes and workflows.
 weight: 25
-keywords: [0.16.x-manage]
+keywords: [0.15.x-manage]
 aliases:
 ---
```

Comment:
General comments for the page (not necessarily at this line):

1) It is standard and a best practice NOT to have tool names inside a shipyard file. Sequences and tasks should be tool agnostic. A shipyard describes **what** you want to achieve, not **how**.
2) External tools (and humans) trigger sequences
3) Tooling respond to task events
4) task properties are optional. `properties` is a mechanism by which you can include properties that the actioning tool requires.